storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -451,6 +451,12 @@
     public static final String TOPOLOGY_METRICS_CONSUMER_REGISTER = "topology.metrics.consumer.register";
 
     /**
+     * Enable tracking of network message byte counts per source-destination task
```

Comment:
It would be good to add this in to defaults.yaml.  Just so it is clear what the default is.  It would also be nice to explain why it is disabled by default.  At least from the code that is what it looks like is the intention.